tests/nose_runner.py

```diff
@@ -69,21 +58,24 @@ def __call__(self):
         if os.path.isfile("{}.xml".format(self.name)):
             os.remove("{}.xml".format(self.name))
         nose.run(argv=self.argv, addplugins=[AnswerTesting()], exit=False)
-        sys.stderr = old_stderr
-        return mystderr.getvalue()
+        return ""
 
     def __str__(self):
         return "WILL DO self.name = %s" % self.name
 
 
 def generate_tasks_input():
     pyver = "py{}{}".format(sys.version_info.major, sys.version_info.minor)
+    if sys.version_info < (3, 0, 0):
+        DROP_TAG = "py3"
+    else:
+        DROP_TAG = "py2"
```

Comment:
@Xarthisius sorry I didn't pay attention to this earlier, may I ask why this change was necessary ? I mean we don't support python 2 at all anymore so I'm a bit confused.

Reply:
It's not necessary. I will drop it